### ISSUE TITLE
User Guide: Document absence of audio ducking facility in temporary copies of NVDA

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#6519]
While testing a recent Next snapshot of NVDA, I was running it as a temporary copy. In the process, I discovered that audio ducking was absent, and upon discussion with an NVDA developer, it has been confirmed that due to technical limitations, audio ducking cannot be made available to temporary copies of NVDA.
In such a case, I sincerely request documenting in NVDA’s User Guide that audio ducking facility is absent in temporary copies of NVDA due to inevitable technicalities, for the knowledge of the user and to avoid any potential confusion.

**Comment by josephsl** [comment]
Hi, this also applies to portable copy as well. CC @MichaelDCurran, @JCSTeh, @feerrenrut, PR (with other docs fixes) on its way. Thanks.

**Comment by josephsl** [comment]
Ref #3830

**Comment by Brian1Gaff** [comment]
There unfortunately as a resent discussion here suggested, the portable copy 
of nvda is far less functional in windows 10 than in earlier versions of 
windows. even programs like Ccleaner has problems with its lists of things 
not displaying in a portable version and some control panel  areas are 
unusable unless you install nvda or use Narrator.
 I suspect this is down to the security requirements of some of the 
accessible aPI components so nasty hackers cannot compromise windows by a 
back door based on the lower level access portable apps might otherwise 
allow in this context..

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by feerrenrut** [comment]
P1 since its important that this is looked at before the next release.
